build/zip_bundle.gni

```diff
@@ -0,0 +1,7 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
What's this file?

Reply:
Meant to be part of a different branch - removed.